src/Backend/Core/Js/ckeditor/plugins/medialibrary/plugin.js

```diff
@@ -16,16 +21,30 @@ CKEDITOR.plugins.add(
                     label: 'Insert Link',
                     toolbar: 'insert,0'
                 }
-            )
+            );
+
+            editor.ui.addButton(
+                'ForkImage',
+                {
+                    command: 'addImage',
+                    // icon: this.path + 'icons/image.png',
+                    label: 'Insert Image',
+                    toolbar: 'insert,0'
+                }
+            );
 
             CKEDITOR.dialog.add('linkDialog', this.path + 'dialogs/linkDialog.js');
+            CKEDITOR.dialog.add('imageDialog', this.path + 'dialogs/imageDialog.js');
 
             // Disable default link dialog behaviour
             editor.on('doubleclick', function (event) {
-                var element = event.data.element
+                var element = event.data.element;
                 if (element.is('a')) {
                     event.data.dialog = 'linkDialog'
                 }
+                if (element.is('img')) {
+                    event.data.dialog = 'imageDialog'
```

Comment:
needs to end with `;`

Reply:
@jonasdekeukelaere JS standards state that you shouldn't use `;`

Reply:
but it might be nice to be consistent ;)